### ISSUE TITLE
iOS: Long titles cover the "users" icon

### ISSUE THREAD

**Issue by whoyawn** [zulip/zulip-mobile#479]
For stream narrows longer than the width of the navigation bar, the right nav button disappears.

![screen shot 2017-04-02 at 7 04 26](https://cloud.githubusercontent.com/assets/15709918/24596214/74e0f396-17f2-11e7-9167-9ba972694bc2.png)

Expected: nav button should still be there
How to reproduce: Switch into any stream title with a long name

**Comment by vishwesh3** [comment]
How about splitting stream (title) and topic (subtitle) in different lines. Using [ellipsizeMode](https://facebook.github.io/react-native/docs/text.html#ellipsizemode) for the topic (subtitle). Or using [MarqueeLabel](https://github.com/remobile/react-native-marquee)

**Comment by timabbott** [comment]
It's possible that just using elipses to cut off the topic is reasonable here (maybe with some way to touch the heading and see the full version); I suspect it's rarely the case the end of the super long topic is important to be able to see in the heading.

**Comment by vishwesh3** [comment]
@zulipbot claim

**Comment by zulipbot** [comment]
Hello @vishwesh3, you claimed this issue to work on it, but this issue and any referenced pull requests haven't been updated for a week. Are you still working on this issue?

If so, please update this issue by leaving a comment on this issue to let me know that you're still working on it. Otherwise, I'll automatically remove you from this issue in 3 days.

If you've decided to work on something else, simply comment `@zulipbot abandon` so that someone else can claim it and continue from where you left off.

Thank you for your valuable contributions to Zulip!

**Comment by zulipbot** [comment]
Hello @vishwesh3, you claimed this issue to work on it, but this issue and any referenced pull requests haven't been updated for a week. Are you still working on this issue?

If so, please update this issue by leaving a comment on this issue to let me know that you're still working on it. Otherwise, I'll automatically remove you from this issue in 3 days.

If you've decided to work on something else, simply comment `@zulipbot abandon` so that someone else can claim it and continue from where you left off.

Thank you for your valuable contributions to Zulip!

**Comment by gnprice** [comment]
Good news! This no longer reproduces. Here's v26.7.130 on an iPhone 7:

<img width="400" src="https://user-images.githubusercontent.com/28173/64287495-ba18e500-cf14-11e9-91e2-6b3360810b6a.png">

The alignment isn't *great*; the ellipsis intrudes a bit visually into the "topic list" icon.  But there is an ellipsis, and the icons show up normally and work normally.

So it seems we've fixed this at some point in the past two years or so, and just neglected to circle back here. Closing now.
